### ISSUE TITLE
hCaptcha Prompt A11Y Evaluation (round 3)

### ISSUE THREAD

**Issue by thiessenp-cds** [cds-snc/platform-forms-client#7811]
The issues identified by AAACT (see #7060) have been resolved by hCaptcha. It's time for another round of testing! 

TODOs
- [x] create a test environment and form that will always show the hCaptcha prompt ([FR test form](https://x6mjsjkclccu5neqdp6lrts2te0abflg.lambda-url.ca-central-1.on.aws/fr/id/cmah86ih60000ic0bq995l9xz), [EN test form](https://x6mjsjkclccu5neqdp6lrts2te0abflg.lambda-url.ca-central-1.on.aws/en/id/cmah86ih60000ic0bq995l9xz))
- [x] request an AAACT a11y audit with the above URL 
- [ ] once the audit is complete, decide what todo next

**Comment by thiessenp-cds** [comment]
I set the audit request deadline for the end of September. Though it could take a few months.

**Comment by anikbrazeau** [comment]
**Update from standup**
- Sent off to AACT, they will complete third re-audit.
- Deadline = one month from now
- Only a couple of issues found, so hopefully not too long to retest
- Once done = accessible hcaptcha challenge!

**Comment by thiessenp-cds** [comment]
There were 7 failures, not including a few from GC Forms. (see attached zip)

I'll send these off to hCaptcha for another iteration.

<img width="1251" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK1QrRGc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--7fcf6607bc60998ad7071cc96726712831bdd041/summary-failures.png" alt="summary-failures.png" />

[2026-09 hcaptcha wcag audit 3.zip](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBK1grRGc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--023871ff83f0f3128ad24e430bd84dfddeed01c1/2026-09%20hcaptcha%20wcag%20audit%203.zip)

**Comment by anikbrazeau** [comment]
**Update from Sep 21 stand-up**
> - hCaptcha is working on the fixes, fix some things and break new things
> - on hold until send new round of fixes to AACT for another round of testing
